XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml

```diff
@@ -74,6 +74,22 @@
         <common:ActivityFeedLayout x:Key="MyFeedLayout" ColumnSpacing="12" RowSpacing="12" MinItemSize="80, 108"/>
         <muxc:UniformGridLayout x:Key="UniformGridLayout2" MinItemWidth="108" MinItemHeight="108" MinRowSpacing="12" MinColumnSpacing="12"/>
 
+        <DataTemplate x:DataType="l:Recipe" x:Key="RecipeTemplate">
+            <StackPanel BorderThickness="1" Background="LightGray" Margin="5">
```

Comment:
switch to one of the existing theme resource. This likely wont look correct in one of the themes.